issuer.go

```diff
@@ -0,0 +1,111 @@
+package issuer
+
+import (
+	"bytes"
+	"compress/flate"
+	"crypto/ecdsa"
+	"encoding/json"
+	"github.com/skip2/go-qrcode"
+	"gopkg.in/square/go-jose.v2"
+	"strconv"
+	"time"
+)
+
+const (
+	MAX_SINGLE_JWS_SIZE = 1195
+	MAX_CHUNK_SIZE      = 1191
+)
+
+type SmartHealthCard struct {
+	IssuerUrl            string                 `json:"iss"`
```

Comment:
Can we use `IssuerURL`? https://github.com/golang/go/wiki/CodeReviewComments#initialisms